src/frontend/test_runner/tests/testdata/agg.yaml

```diff
@@ -657,3 +659,106 @@
         LogicalProject { exprs: [t.x, t.y, flag] }
           LogicalExpand { column_subsets: [[t.x], [t.y]] }
             LogicalScan { table: t, columns: [t.x, t.y] }
+- sql: |
+    /* agg filter: subquery */
+    /* This case is valid in PostgreSQL */
+    create table a (a1 int, a2 int);
+    select count(a1) filter (where (select true)) from a;
+  binder_error: 'Invalid input syntax: subquery in filter clause is not supported'
```

Comment:
Should we return NotImplemented in this case?

Reply:
good catch